.github/workflows/release.yaml

```diff
@@ -28,10 +28,10 @@ jobs:
 
       # https://github.com/helm/chart-releaser-action
       - name: Run chart-releaser
-        uses: helm/chart-releaser-action@v1.5.0
+        uses: helm/chart-releaser-action@v1.6.0
         with:
           charts_dir: deploy/charts
-          charts_repo_url: https://kubeagi.github.io/arcadia/
```

Comment:
will this change the chart repo url if not set ?

Reply:
> Warning: Unexpected input(s) 'charts_repo_url', valid inputs are ['version', 'config', 'charts_dir', 'install_dir', 'install_only', 'skip_packaging']

https://github.com/kubeagi/arcadia/actions/runs/6920106990/job/18824225331

In fact, even in the old helm/chart-releaser-action@v1.5.0 version, this parameter is useless.